src/containers/Header/NotificationsTray/index.jsx

```diff
@@ -34,16 +40,146 @@ class NotificationsTray extends PureComponent {
     this.setState({ viewingNotification: null });
   }
 
+  _onMarkAsReadClick() {
+    if (! this.state.markingMode) {
+      this.setState({
+        markingMode: true,
+      });
+    }
+  }
+
+  _onCancelClick() {
+    this.setState({
+      markingMode: false,
+    });
+  }
+
+  _onMarkClick() {
+    console.log('dega', this.state.notificationIds);
+    this.setState({
+      notificationIds: [],
```

Comment:
Could this name be a bit more descriptive? Something like `notificationIdsToMarkRead`